app/service_routes.js

```diff
@@ -95,4 +102,51 @@ routes.options('/services/images', cors())
  */
 routes.get('/services/images', cors(), jwtCheck, resources.services.images.get)
 
+/**
+ * @swagger
+ * /services/votes:
+ *   get:
+ *     description: Returns a candidate street for the user to vote on
+ *     tags:
+ *       - sentiment
+ *     produces:
+ *      - application/json
+ *     responses:
+ *       200:
+ *         description: Street data
+ *         schema:
+ *           type: object
+ *           properties:
+ *             data:
+ *               type: string
+ */
+routes.get('/services/votes', cors(), jwtCheck, resources.services.votes.get)
+/**
+ * @swagger
+ * /services/votes:
+ *   post:
+ *     description: Adds a user's vote on a particular street
+ *     tags:
+ *       - sentiment
+ *     produces:
+ *      - application/json
+ *     parameters:
+ *       - name: vote
+ *         description: user's score of a street
+ *         in:  body
+ *         required: true
+ *         type: string
```

Comment:
Is this consistent with the API as implemented? From reading the code it seems that street data and userId are also required parameters. Also, will the score itself be a number, or does that get sent as a string?